rest/changes_api.go

```diff
@@ -446,7 +452,7 @@ func generateBlipSyncChanges(database *db.Database, inChannels base.Set, options
 
 	// Store one-shot here to protect
 	isOneShot := !options.Continuous
-	err, forceClose = generateChanges(database, inChannels, options, docIDFilter, nil, send)
+	err, forceClose = generateChanges(context.Background(), database, inChannels, options, docIDFilter, send)
```

Comment:
It feels like we should (at some point?) be passing a context associated with the blipSyncContext, but it doesn't look like that exists today, other than the associated database.Ctx.  Something for a future enhancement?

Reply:
Agreed - will change to `context.TODO()` and add a note